pagopa/io-app-design-system
# docs: [IOAPPX-251] Update README.md

## Short description
This PR updates the `README.md` by including instructions about wrapping the app with `SafeAreaProvider` from `react-native-safe-area-context`. This is needed because a lot of components use the `useSafeAreaInsets` hook which leads to runtime errors if they are being used without the safe area context.

There's no need to install the dependency directly as it is already being declared as `peerDependency`, thus the consumer should be warned about its absence, eventually.

## How to test
Check the new README.md preview [here](https://github.com/pagopa/io-app-design-system/blob/IOAPPX-251-update-readme/README.md).
